src/rpc/fees.cpp

```diff
@@ -96,6 +99,59 @@ static RPCHelpMan estimatesmartfee()
     };
 }
 
+static RPCHelpMan estimatefeewithforecasters()
+{
+    return RPCHelpMan{
+        "estimatefeewithforecasters",
+        "\nEstimates the approximate fee per kilobyte needed for a transaction to begin\n"
+        "confirmation within conf_target blocks if possible Uses virtual transaction size as defined\n"
+        "in BIP 141 (witness data is discounted).\n",
+        {
+            {"conf_target", RPCArg::Type::NUM, RPCArg::Optional::NO, "Confirmation target in blocks"},
+        },
+        RPCResult{
+            RPCResult::Type::OBJ, "", "", {
+                {RPCResult::Type::NUM, "Low priority fee rate estimate (" + CURRENCY_UNIT + "/kvB)", /*optional=*/true, "fee rate estimate in " + CURRENCY_UNIT + "/kvB for economical users"},
+                {RPCResult::Type::NUM, "High priority fee rate estimate (" + CURRENCY_UNIT + "/kvB)", /*optional=*/true, "fee rate estimate in " + CURRENCY_UNIT + "/kvB for conservative users"},
+                {RPCResult::Type::STR, "Forecaster", /*optional=*/true, "the forecaster that provide the fee rate estimate"},
+                {RPCResult::Type::ARR, "errors", /*optional=*/true, "Errors encountered during processing (if there are any)", {
```

Comment:
In  [rpc]: create an rpc estimatefeewithforecasters :

suggest changing to e.g.

```suggestion
                {RPCResult::Type::NUM, "low", /*optional=*/true, "fee rate estimate in " + CURRENCY_UNIT + "/kvB for economical users"},
                {RPCResult::Type::NUM, "high", /*optional=*/true, "fee rate estimate in " + CURRENCY_UNIT + "/kvB for conservative users"},
```

as per Signal convo

Reply:
Taken, but I left out the `forecaster` and `errors` from the outputs for now.

Also, per our conversation, I looked into why the block value is returned in `estimatesmartfee`. It's because sometimes you may want to estimate the fee for a 5-confirmation target, but there may be no valid estimate for that target. In such cases, it might return an estimate for a different block target like `6` where a valid estimate is available.

This does not apply to our forecasters.